container/shim/src/utils/jwt.js

```diff
@@ -0,0 +1,23 @@
+import jwt from "jsonwebtoken";
+
+export function findJWT(req) {
+  const jwtQuery = req.variables.arg_jwt;
+
+  let jwtHeader = "";
+  const authHeader = req.variables.http_authorization;
+  if (authHeader) {
+    jwtHeader = authHeader.replace("Bearer ", "");
+  }
+
+  return jwtQuery || jwtHeader;
+}
```

Comment:
This looks like njs code? This won't work in express.

Reply:
good eyes Eric, changed.